src/palette/view/widgets/timedialog.cpp

```diff
@@ -54,7 +54,7 @@ TimeDialog::TimeDialog(QWidget* parent)
     sp->setReadOnly(false);
     sp->setSelectable(true);
 
-    connect(zNominal, &QSpinBox::valueChanged, this, &TimeDialog::zChanged);
+    connect(zNominal,  &QSpinBox::editingFinished, this, &TimeDialog::zChanged);
```

Comment:
Unnecessary whitespace change:
```suggestion
    connect(zNominal, &QSpinBox::editingFinished, this, &TimeDialog::zChanged);
```